homeassistant/components/bond/utils.py

```diff
@@ -58,31 +58,39 @@ def trust_state(self) -> bool:
         """Check if Trust State is turned on."""
         return self.props.get("trust_state", False)
 
+    def _has_any_action(self, actions: Set[str]):
```

Comment:
Missing return value typing.

If this integration has almost complete typing coverage I'd recommend enforcing that in our CI by updating the config line in `setup.cfg`.

https://github.com/home-assistant/core/blob/66027bcef5a0f8d425ebb64ded7339eaa4fef72a/setup.cfg#L45

Reply:
done in https://github.com/home-assistant/core/pull/47187